src/span_context.js

```diff
@@ -98,6 +98,20 @@ export default class SpanContext {
     return this._spanIdStr;
   }
 
+  get toTraceId(): ?string {
```

Comment:
* isn't OpenTracing defining toSpanId as always returning a string? So it should not be `?string`, just `string`. To match w/ `spanIdStr` which is `?string`, you can `return this.spanIdStr || "";`
* iirc it should be a function, not a property, so no `get`.

Reply:
You're absolutely right. Let me make the changes and send it again